webui/react/src/components/TagList.tsx

```diff
@@ -125,16 +95,14 @@ const EditableTagList: React.FC<Props> = (
               <Input
                 aria-label={ARIA_LABEL_INPUT}
                 className={css.tagInput}
+                defaultValue={tag}
                 key={tag}
                 ref={editInputRef}
                 size="small"
                 style={{ width: inputWidth }}
-                value={editInputValue}
                 width={inputWidth}
-                onBlur={handleEditInputConfirm}
-                onChange={handleEditInputChange}
-                onFocus={handleEditInputFocus}
-                onPressEnter={handleEditInputConfirm}
```

Comment:
nice 👍 much cleaner! Tried out the updated taglist via preview, this seems to work without any issues too.